doc/release/0.17.0-notes.rst

```diff
@@ -37,6 +37,17 @@ are deprecated. These are related to ANOVA, for which ``scipy.stats`` provides
 quite limited functionality and these functions are not very useful standalone.
 See issues #660 and #650 for details.
 
+``scipy.stats.chisqprob`` is deprecated. This is a delegator function and it is
+deprecated in favor of the convention, which is to call the distribution from 
+``scipy.stats.distribution`` or calling ``scipy.special.chdtrc`` directly.
+See issues #2741 and #641 for details.
```

Comment:
Same comments as for ``betai``, except that this does have a dist method: it's simply ``stats.chi2.sf`` (which is the recommended way to write it, not ``chdtrc``).